packages/react-i18n/src/currencyCode.ts

```diff
@@ -17,7 +17,6 @@ export enum CurrencyCode {
   Bsd = 'BSD',
   Bhd = 'BHD',
   Bif = 'BIF',
-  Byn = 'BYN',
```

Comment:
These got shifted around in the file, rather than removed, because I did a straight copy-paste from `types/graphql/core-types` in `Shopify/web`, which generated the type from `Shopify/shopify`.